.github/workflows/CI.yaml

```diff
@@ -24,63 +24,59 @@ jobs:
     strategy:
       matrix:
         os: [macOS-latest, ubuntu-latest, windows-latest]
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
         # Only test lowest and highest version on the expensive/slow
         # macOS and windows runners (UPDATE when supported versions change):
         exclude:
-        - os: macOS-latest
-          python-version: 3.8
-        - os: macOS-latest
-          python-version: 3.11
-        - os: windows-latest
-          python-version: 3.8
-        - os: windows-latest
-          python-version: 3.11
+          - os: macOS-latest
+            python-version: 3.10
+          - os: macOS-latest
+            python-version: 3.11
+          - os: windows-latest
+            python-version: 3.10
+          - os: windows-latest
+            python-version: 3.11
 
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Setup python
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
-
-    - name: Display Python version
-      run: python -c "import sys; print(sys.version)"
-
-    - name: Additional info about the build
-      shell: bash
-      run: |
-        uname -a
-        df -h
-        ulimit -a
-
-    - name: Install pytest, pytest plugins
-      run: |
-        python -m pip install wheel
-        python -m pip install pytest pytest-cov pytest-pep8
-
-    - name: Install package
-      run: |
-        python -m pip install .
-
-    - name: Run tests
-
-      # conda setup requires this special shell
-      shell: bash -l {0}
-
-      run: |
-        pytest -v --cov=alchemtest --cov-report=xml --color=yes src/alchemtest/tests
-
-    - name: CodeCov
-      uses: codecov/codecov-action@v3
-      with:
-        token: ${{ secrets.CODECOV_TOKEN }}
-        name: codecov-${{ matrix.os }}-py${{ matrix.python-version }}      
-        file: ./coverage.xml
-        flags: unittests
-        fail_ci_if_error: true
-
+      - uses: actions/checkout@v3
+
+      - name: Additional info about the build
+        shell: bash
+        run: |
+          uname -a
+          df -h
+          ulimit -a
```

Comment:
We can probably delete this section — I don't think I ever cared about this info.